aws-codegurureviewer-repositoryassociation/src/main/java/software/amazon/codegurureviewer/repositoryassociation/Translator.java

```diff
@@ -55,6 +60,14 @@ static AssociateRepositoryRequest translateToAssociateRepositoryRequest(final Re
             throw new CfnInvalidRequestException(String.format("Unknown Type of %s", model.getType()));
         }
 
+        final Optional<Map<String, String>> tags = getTagsFromModel(model);
+        if(tags.isPresent()) {
+            return AssociateRepositoryRequest.builder()
+                    .repository(repository)
+                    .tags(tags.get())
+                    .build();
+        }
+
         return AssociateRepositoryRequest.builder().repository(repository).build();
```

Comment:
`tagsOptional.map( "with tags").orElse("without tags")`

this is much more readable with the usage of Optional.

Reply:
I agree Optional.map().orElse() is a more readable usage. But I think it is not quite fit into the above situation.